content/advocate-with-mau/be-a-member.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Catch 'Em Predator
+description: If you find our work resonating with you, join the MAU team as a member today, you will receive exclusive opportunities for capacity building,leading projects, networking with subject matter experts in child protection, and also make friends with people who are also passionate about the cause.
+image: /img/advocate/forum.jpg
+carousel:
+  - image: /img/advocate/activity.jpg
+  - image: /img/advocate/discuss.jpg
+  - image: /img/advocate/forum.jpg
+  - image: /img/advocate/together.jpg
+---
+
+What are you waiting for? Just click the link below!  
```

Comment:
For this content, I think its best to make it static. Move the link into the frontmatter